blc/checker.py

```diff
@@ -101,33 +103,31 @@ def check(self, url: str) -> requests.Response:
         :url represent the URL to check
         """
         # We verify the URL is already checked
-        if url in self.checked_url:
+        if [u for u in self.urls if self.urls[u]['result'] and url == u]:
```

Comment:
```suggestion
        if url in self.urls.keys() and self.urls[url].get('result', False):
```
What do you think about this instead of looping?